hermes-common/src/main/java/pl/allegro/tech/hermes/common/metric/counter/zookeeper/CounterMatcher.java

```diff
@@ -40,6 +45,10 @@ public boolean isSubscriptionInflight() {
         return counterName.startsWith("inflight.");
     }
 
+    public boolean isSubscriptionFiltered() {
+        return counterName.startsWith("filtered.");
```

Comment:
added this to send filtered counter to ZK instead of graphite - i assume this was a bug.